Resources/Lang/en_US.json

```diff
@@ -887,19 +887,19 @@
   "ProvocateurInfoLong": "(Neutrals):\nAs the Provocateur, you can kill any target with the kill button. If the target loses at the end of the game, the Provocateur wins with the winning team.",
   "BloodKnightInfoLong": "(Neutrals):\nThe Blood Knight wins when they're the last killing role alive, and the amount of crewmates is lower or equal to the amount of Blood Knights. The Blood Knight gains a temporary shield after every kill, making them immortal for a few seconds.",
   "PlagueBearerInfoLong": "(Apocalypse):\nAs the Plaguebearer, plague everyone using your kill button to turn into Pestilence.\nOnce you turn into Pestilence, you will become immortal and gain the ability to kill, and you will kill anyone who tries to kill you.\n\nAlso, when infected players interact with uninfected players, they will also be infected.",
-  "PestilenceInfoLong": "(Apocalypse):\nAs Pestilence, you're an unstoppable machine.\nAny attack towards you will be reflected towards them.\nIndirect kills don't even kill you.\n\nOnly way to kill Pestilence is by vote, or by it misguessing.\n<b>Your presence is announced to everyone at the meeting after you transform.</b>",
+  "PestilenceInfoLong": "(Apocalypse):\nAs Pestilence, you're an unstoppable machine.\nAny attack towards you will be reflected towards them.\nIndirect kills don't even kill you.\n\nOnly way to kill Pestilence is by vote or by misguessing.\n<b>Your presence is announced to everyone at the meeting after you transform.</b>",
```

Comment:
This indicates the non-Pestilence player may misguess to exile Pestilence. Either reword it or re-add "it"